### DISCUSSION TITLE
Not starting

### DISCUSSION THREAD

**Discussion by NotOriqi** [Alex313031/thorium#371] [Q&A] [answered]
i installed the mini installer for windows and it installed and i tried launching thorium and it wouldnt start it would start for like a second then it will close itself

**Comment by gz83** [comment, marked as answer]
please use this version

https://github.com/Alex313031/Thorium-SSE3/releases/tag/M117.0.5938.157-1

**Comment by NotOriqi** [comment]
Thank You it works now
